### DISCUSSION TITLE
Units in MOOSE

### DISCUSSION THREAD

**Discussion by agniksr** [idaholab/moose#28128] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

So I am trying to solve the heat conduction equation on an annular cylinder made out of zircaloy-2. I have in SI units the various parameters required, such as specific heat, density, thermal expansion coefficient and thermal conductivity. However, my mesh dimensions are in centimeters and not meters, so should i change all my parameters in CGS units or should i just convert the length part, or is there something else i should do

**Comment by GiudGiud** [comment]
Hello
You can use a transformMeshGenerator to scale your mesh back to meters

Use the FileMeshGenerator to read your original mesh and set it as the input to the transformMeshGenerator

Guillaume

**Reply by agniksr** [reply]
Hey 
What I meant was that my mesh is a annular pipe which needs to be in centimeters (like 7.5cm inner radius and 10cm outer radius).
So do i need to modify the values of the constants like converting them or something like that

**Reply by GiudGiud** [reply]
You need to be consistent between the mesh units and the units in use in the simulation
Either scale the mesh or scale the constants.
